README.md

```diff
@@ -12,7 +12,7 @@
 </a>
 
 <p>Cheatsheets for experienced React developers getting started with TypeScript</p>
-
+ 3
```

Comment:
accident?

Reply:
```suggestion
```

Reply:
Just testing if you're still sane after going through 1000+ notifications :smile: